src/test/java/uk/gov/companieshouse/overseasentitiesapi/interceptor/UserAuthenticationInterceptorTest.java

```diff
@@ -75,6 +88,75 @@ void testInterceptorReturnsFalseWhenRequestHasIncorrectTokenPermission() {
         assertEquals(HttpServletResponse.SC_UNAUTHORIZED, mockHttpServletResponse.getStatus());
     }
 
```

Comment:
As mentioned above, please add a new test for when the call to retrieve the transaction fails and check that the response status code is 500 (internal server error).